### ISSUE TITLE
The IDE cannot execute a test script in the directory. Possible reason: the partition is mounted with 'no exec' option.

### ISSUE THREAD

**Issue by sanecito** [flathub/com.jetbrains.PyCharm-Community#535]
**Problem:**

When launching PyCharm Community the following error dialogue popup occurs:

```
The IDE cannot execute a test script in the directory.
Possible reason: the partition is mounted with 'no exec' option.

If you have modified the 'idea.system.path' property, please make sure it is correct,
otherwise, please re-install the IDE.

-----
Location: /home/GoodUser/.var/app/com.jetbrains.PyCharm-Community/cache/JetBrains/PyCharmCE2023.1/tmp
java.io.IOException: Unexpected exit value: 134
```
Selecting "OK" removes the error dialog, but does not continue onwards to PyCharm-Community

**Expectation:**
Able to launch PyCharm Community, preferably without error dialogs

**Software Info:**
Operating System: Fedora Kinoite Linux 37
KDE Plasma Version: 5.27.4
KDE Frameworks Version: 5.104.0
Qt Version: 5.15.8
Kernel Version: 6.2.9-200.fc37.x86_64 (64-bit)
Graphics Platform: Wayland
PyCharm-Community: 2023.1

**Comment by Saturn745** [comment]
Same problem

**Comment by bademux** [comment]
the same for IntelliJ Idea (Java)  - its broken.

**Comment by bademux** [comment]
Seems to be know problem, unfortunately it was released without much testing  
https://github.com/flathub/com.jetbrains.PyCharm-Community/pull/534#issuecomment-1501811242

**Comment by alex27riva** [comment]
I have the same issue on Fedora 38 Silverblue.

**Comment by rdbeni0** [comment]
the same

**Comment by huaracheguarache** [comment]
This still hasn't been fixed in version 2023.1.2.
